### DISCUSSION TITLE
How do you start mlflow with mlflow-oidc-auth ?

### DISCUSSION THREAD

**Discussion by lguichard78** [mlflow-oidc/mlflow-oidc-auth#39] [Q&A] [answered]
Hi,
What is the correct command to start the server ? Is the one below correct ?
$ mlflow ui --app-name oidc-auth 
Best regards

**Comment by kharkevich** [comment, marked as answer]
To run tracking server you can use the following command:
```sh
mlflow server --app-name oidc-auth --host=0.0.0.0 --port=8080 --backend-store-uri=sqlite:///:memory --default-artifact-root=./mlruns --serve-artifacts
```

Additional example you can find here:
- https://github.com/data-platform-hq/mlflow-oidc-auth/blob/main/scripts/run-dev-server.sh

Or in mlflow documentation:
- https://mlflow.org/docs/latest/tracking/server.html
- https://www.mlflow.org/docs/latest/auth/index.html
